src/api/namespaces/atomicmarket/handlers/sales.get-sales.test.ts

```diff
@@ -177,42 +177,28 @@ describe('AtomicMarket Sales API', () => {
                 .to.deep.equal([sale_id]);
         });
 
-        txit('throws error when maximum price filter is set without settlement symbol', async () => {
+        txit('filters by minimum price', async () => {
             let err;
             try {
-                await getSalesIds({max_price: '1'});
+                await getSalesIds({min_price: '2'});
             } catch (e) {
                 err = e;
             }
 
             expect(err).to.be.instanceof(ApiError);
-            expect(err.message).to.equal('Price range filters require the "symbol" filter');
-        });
-
-        txit('filters by minimum price', async () => {
-            await client.createSale({});
-
-            const {sale_id} = await client.createSale({
-                listing_price: 200000000,
-            });
-
-            await client.query('REFRESH MATERIALIZED VIEW atomicmarket_sale_prices');
-
-            expect(await getSalesIds({symbol: 'TEST', min_price: '2'}))
-                .to.deep.equal([sale_id]);
+            expect(err.message).to.equal('Price filters are removed in /v1/sales, use /v2/sales');
```

Comment:
Maybe instead of testing for the message, extend the API error and create a DeprecatedFilter exception and with that you just dont need to assert the message.

Later if the message changes this text will start to fail for no reason

Reply:
Well I think you should assert the message either way, since it's not a generic error